syntax = "proto3";

 option java_multiple_files = true;
 option java_package = "environmentService";
 option java_outer_classname = "EnvironmentServiceImpl";

  //package environmentService
 package environmentService;

 service EnvironmentService {

 	//Unary call
	rpc setRoomTemp(TempRequest) returns(TempResponse){}

 	//Unary call
	rpc getCurrentRoomTemp(Empty) returns(CurrentResponse){}
 }

 message TempResponse{
	string result = 1;
 }

 message CurrentResponse{
	int32 value = 1;
 }

 //Unary call
 message TempRequest{
	int32 value = 1;
 }

 message Empty{

 }